changelog/issue-2962.md

```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: minor
```

Comment:
Why minor?  Since this isn't even used in this version, I'd think 'silent'.  PRs adding new database versions will be minor, so there will be visibility in the version..